libs/designsystem/accordion/src/accordion-item.component.spec.ts

```diff
@@ -72,4 +80,12 @@ describe('AccordionItemComponent', () => {
       });
     });
   });
+  it('should emit the toggle event when collapsed', () => {
+    spyOn(spectator.component.toggle, 'emit');
+    spectator.component.isDisabled = false;
+    spectator.component.isExpanded = true;
+    spectator.component._onToggleExpanded(new KeyboardEvent('keydown'));
```

Comment:
Consider using `spectator.click` method instead to actually test that clicking the item triggers the event (calling `_onToggleExpanded` explicitly doesn't test that the method is wired up to the markup):
```suggestion
    spectator.click('.header');
```